theme/static/css/hydroshare_core.css

```diff
@@ -4819,6 +4819,7 @@ div.property-selector {
 
 .full-width {
   width: 100%;
+  word-break: break-word;
```

Comment:
@devincowan - Not to be too picky, but I think this rule should be its own utility class and use where appropriate in order to prevent undesired/unexpected effects.

Reply:
We could try to mimic Bootstrap 4 implementation of this utility class in order to keep our code consistent: https://getbootstrap.com/docs/4.3/utilities/text/#word-break

Reply:
sounds good, added a utility class, thanks